aspnet/migration/rc2-to-rtm.rst

```diff
@@ -0,0 +1,56 @@
+Migrating from ASP.NET Core RC2 to ASP.NET Core 1.0
+===================================================
+
+By `Cesar Blum Silveira`_
+
+.. contents:: Sections:
+  :local:
+  :depth: 1
+
+Overview
+--------
+
+This migration guide covers migrating an ASP.NET Core RC2 application to ASP.NET Core 1.0.
+
+There weren't many significant changes to ASP.NET Core between the RC2 and 1.0 releases. For a complete list of changes, see the `ASP.NET Core 1.0 announcements <https://github.com/aspnet/announcements/issues?q=is%3Aopen+is%3Aissue+milestone%3A1.0.0>`_.
+
+Tools
+-----
+
+You no longer need to use ``imports`` for tools in ``project.json``. You can change, for example, the following entry:
+
+.. code-block:: json
+
+  {
+    "tools": {
+      "Microsoft.AspNetCore.Server.IISIntegration.Tools": {
+        "version": "1.0.0-preview1-final",
+        "imports": "portable-net45+win8+dnxcore50"
+      }
+    }
+  }
+
+To:
+
+.. code-block:: json
+
+  {
+    "tools": {
+      "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-preview2-final"
+    }
+  }
+
+Hosting
+-------
+
+The ``UseServer`` is not longer available for :dn:iface:`~Microsoft.AspNetCore.Hosting.IWebHostBuilder`. You must now use :dn:method:`~Microsoft.AspNetCore.Hosting.WebHostBuilderKestrelExtensions.UseKestrel` or :dn:method:`~Microsoft.AspNetCore.Hosting.WebHostBuilderWebListenerExtensions.UseWebListener`.
+
+MVC
+---
+
+The :dn:class:`~Microsoft.AspNetCore.Html.HtmlString` class has been removed from MVC. You should now use the :dn:class:`~Microsoft.AspNetCore.Html.HtmlString` class provided in the ``Microsoft.AspNetCore.Html.Abstractions`` package, which was formerly named ``HtmlEncodedString``. If you were using :dn:class:`~Microsoft.AspNetCore.Html.HtmlString` directly in your Razor views, you must add a ``@using`` statement for :dn:namespace:`Microsoft.AspNetCore.Html`.
```

Comment:
I'm not sure it makes sense to use a "type" link to a type that no longer exists, because presumably it'll be a 404...